PageObjectPattern/src/test/java/api/stepdefinitions/SecondTestStepDefinitions.java

```diff
@@ -0,0 +1,24 @@
+package api.stepdefinitions;
+
+import apiSerenityStep.AppAuthStep;
+import apiSerenityStep.TestBaseStep;
+import io.cucumber.java.en.*;
+import net.thucydides.core.annotations.Steps;
+
+public class SecondTestStepDefinitions {
+
+    @Steps
+    TestCommonStepDefinitions commonStep;
+
+    @Steps
+    AppAuthStep auth;
+
+    @Steps
+    TestBaseStep step;
+
+    @When("user sends request to categories endpoint")
+    public void getRequest() {
+        String endpoint = "/v1/browse/categories";
```

Comment:
Same as above, the endpoint itself can be moved to the REST request either directly into REST-assured method .get() or by creating a class constant of TestBaseStep.